insights/specs/default.py

```diff
@@ -548,6 +548,39 @@ def semid(broker):
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
     ovirt_engine_boot_log = simple_file("/var/log/ovirt-engine/boot.log")
     ovirt_engine_console_log = simple_file("/var/log/ovirt-engine/console.log")
+    ovs_vsctl_list_br = simple_command("/usr/bin/ovs-vsctl list-br")
+
+    @datasource(ovs_vsctl_list_br)
```

Comment:
I think this should be a `foreach_execute` with `ovs_vsctl_list_br` passed to it. Calling `/usr/bin/ovs-appctl fdb/show` manually for each bridge name and then putting all of the results into a single response means there will be only a single parser instance, and that's inconsistent with the expectations we've set with other parsers that represent multiple things (ethtool, etc.).

If we have to choose between inconsistency by collecting now with a method that we'll eventually remove and consistency by waiting until core3 collection on the client, we should do the latter.

Reply:
@csams - Yeap, we have considered `foreach_execute` at the very beginning. The reason for not using it is:

1. The output of `ovs-appctl fdb/show bridge_name` contains MAC addresses, it's PII data that might be not allowed to collect, but the plugin only cares about their count.
   
2. We thought the `foreach_execute` might cannot be collected in the near future, which results in the plugin has no hit. But the `glob_file` can simply resolve this problem by collecting all the `ovs-appctl_fdb_show_*` files.

If it's OK to collect MAC addresses, I think it's also OK to refactor this spec to `foreach_execute`.

Reply:
@xiangce @csams - As per suggestions, I have made the following changes:
* Refactor this spec to use `foreach_execute` instead of `datasource`.
* Modify the parser to parse the complete output of command `/usr/bin/ovs-appctl fdb/show [bridge-name]`.